### PR TITLE
Remove Images as a test dep

### PR DESCRIPTION
**Pull request by ianshmean**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -5,12 +5,10 @@ version = "0.1.2"
 
 [deps]
 CEnum = "fa961155-64e5-5f13-b03f-caf6b980ea82"
-ImageCore = "a09fc81d-aa75-5fe9-8630-4744c3626534"
```
Comment (Drvi) [inline Project.toml]: We still need `FixedPointNumbers` and `Colors` + the [transpose definitions from ImageCore](https://github.com/JuliaImages/ImageCore.jl/blob/65b42543fc90e676eb306db5c0d642b1d8a7617f/src/ImageCore.jl#L152).

File: src/io.jl [hunk 2]
```diff
@@ -293,14 +293,11 @@ function __prepare_buffer(x::AbstractArray{T,3}) where {T}
...
+        GrayA.(view(x, :, :, 1),view(x, :, :, 2))
     elseif nchannels == 3
-        convert(Array{RGB}, colorview(RGB, view(x, :, :, 1), view(x, :, :, 2), view(x, :, :, 3)))
+        RGB.(view(x, :, :, 1), view(x, :, :, 2), view(x, :, :, 3))
     elseif nchannels == 4
-        convert(
-            Array{RGBA},
-            colorview(RGBA, view(x, :, :, 1), view(x, :, :, 2), view(x, :, :, 3), view(x, :, :, 4)),
-        )
+        RGBA.(view(x, :, :, 1), view(x, :, :, 2), view(x, :, :, 3), view(x, :, :, 4))
```
Comment (Drvi) [inline src/io.jl]: This is much nicer :+1:

**Comment by Drvi** [comment]
If we add [maxabsfinite](https://github.com/JuliaPlots/VisualRegressionTests.jl/blob/master/src/utils.jl#L58) to utils, we can depend on ImageCore only in tests (no need for full blown Images)

**Comment by ianshmean** [comment]
> If we add maxabsfinite to utils, we can depend on ImageCore only in tests (no need for full blown Images)
Yeah, I thought about that too. I'll give it a go

File: src/io.jl (hunk 2 above)
Comment (Drvi) [reply src/io.jl]: On the other hand, if we'll keep the ImageCore maybe the original version was a little bit faster:
```
julia> g(x) = convert(
                   Array{RGBA},
                   colorview(RGBA, view(x, :, :, 1), view(x, :, :, 2), view(x, :, :, 3), view(x, :, :, 4)),
              )
julia> f(x) = RGBA.(view(x, :, :, 1), view(x, :, :, 2), view(x, :, :, 3), view(x, :, :, 4))

julia> img = rand(N0f16, 2048, 2048, 4)

julia> @btime g(img);
  43.988 ms (8 allocations: 32.00 MiB)

julia> @btime f(img);
  52.528 ms (6 allocations: 32.00 MiB)

julia> img = rand(N0f16, 64, 64, 4)

julia> @btime g(img);
  39.093 μs (8 allocations: 32.41 KiB)

julia> @btime f(img);
  39.612 μs (6 allocations: 32.33 KiB)

```

Comment (ianshmean) [reply src/io.jl]: Thanks for testing that! Yeah, I'll revert given I've switched this PR to just removing Images.jl from the tests

Comment (kimikage) [reply src/io.jl]: Although I have not benchmarked it, a simple `for` loop may be faster.

**Pull request merged**